### PR TITLE
fix: keep machine config on unconfigure

### LINKED ISSUE #1198: VxPrint - unloading / reloading election switches machine back to VxMark mode

### PR DESCRIPTION
**Pull request by eventualbuddha**
Closes #1198

### CODE REVIEW

File: src/AppPrintOnly.test.tsx [hunk 1]
```diff
@@ -396,3 +397,106 @@ it('VxPrintOnly flow', async () => {
   // Default Unconfigured
   getByText('Device Not Configured')
 })
+
+test('VxPrint retains configuration after unload', async () => {
```
Comment (benadida) [inline src/AppPrintOnly.test.tsx]: let's not call it "configuration", it should only retain machine type and machine ID.

Comment (eventualbuddha) [reply src/AppPrintOnly.test.tsx]: Hardware configuration?

Comment (eventualbuddha) [reply src/AppPrintOnly.test.tsx]: Hardware state?

Comment (eventualbuddha) [reply src/AppPrintOnly.test.tsx]: How about "VxPrint retains app mode when unconfigured"?

File: src/AppPrintOnly.test.tsx [hunk 2]
```diff
@@ -396,3 +397,106 @@ it('VxPrintOnly flow', async () => {
...
+
+  // Remove election configuration.
+  await unconfigure()
+
+  // Re-configure & open polls again.
+  await configure()
+  await openPolls()
+
+  // Make sure we're again ready to print ballots.
+  getByText('Insert Card to print your official ballot.')
```
Comment (beausmith) [inline src/AppPrintOnly.test.tsx]: Suggestion: While we're abstracting to functions, this (and the line above) could be appended to the `openPolls` function.

Comment (eventualbuddha) [reply src/AppPrintOnly.test.tsx]: I know, but I wanted it to be very clear what this test is checking.

**Pull request merged**